spec/unit/code_climate_spec.rb

```diff
@@ -0,0 +1,11 @@
+RSpec.describe Nib::CodeClimate do
+  subject { described_class.new(nil, []) }
+
+  before do
+    allow(subject).to receive(:system)
+  end
+
+  it 'runs the codeclimate docker image' do
+    expect(subject.script).to match(/docker-compose.*run.*--rm.*codeclimate/)
```

Comment:
do we want to make sure the command is getting passed to codeclimate? e.g. test with the analyze command?